warehouse/README.md

```diff
@@ -38,3 +39,30 @@ docker-compose up -d
 
 Then use any db client to connect to `localhost:5431` using the db credentials
 in `.env`.
+
+## Data Organization
+
+This warehouse currently stores the cleaned and merged source data from Kids First studies which have been ingested using the ingest [library](https://github.com/kids-first/kf-lib-data-ingest) and [packages](https://github.com/kids-first/kf-ingest-packages).
+
+The cleaned source data tables are produced by ingest library's extract stage and the merged data tables are produced by the library's transform stage. Any column name in any of these tables will be a concept attribute from the library's [standard concept schema](https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/common/concept_schema.py).
```

Comment:
This is my fault. The standard concept schema markdown link doesn't render properly. Probably because the actual hyperlink needs to all be on one line. Can you fix it?

Reply:
The hyperlink seems working fine. What's the issue?

Reply:
Oh weird it is working now

Reply:
Oh, you mean the white space between [] and ()? I removed it :)